src/PageTransition.jsx

```diff
@@ -214,6 +226,9 @@ export default class PageTransition extends React.Component {
                 Promise.resolve()
               );
             }
+            if(this.props.transitionAction && this.props.transitionAction != ""){
```

Comment:
We can move this part `this.props.transitionAction && this.props.transitionAction != ""` to a method, it will reduce duplicate code: `if (this.hasTransitionAction()) { ... }`.

I'll checkout the branch and do some local testing. If everything is ok I'll merge this tomorrow morning (it's late at night here).

Reply:
Ok kwl. If you want to change the formatting that's fine, the functionality is there and works fine in my testing.